index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Benson Portfolio</title>
+</head>
+<body>
+    <header id="header">
+        <section id="headertoolbar">
+            <div id="headerlogo">My Logo</div>
+            <nav class="navheader">
+                <button id="headermenubtn" type="button"><img src="/images/Union.png" alt="dropdown menu"></button>
+                <div class="desktop">
+                    <a href="#works">Portfolio</a>
+                    <a href="#about">About</a>
+                    <a href="#contact">Contact</a>
+                </div>
+            </nav>
+        </section>
+        <article id="headerarticle">
+            <h1>I’m Benson.<br>Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <p class="connect">LET’S CONNECT</p>
+            <ul class="headericons">
+                <li><a href=""><img src="images/Twitter icon.png" alt="twitter"></a></li>
+                <li><a href="www.linkedin.com/in/bensonkiai"><img src="images/Linkedin icon.png" alt="linkedin"></a></li>
+                <li><a href=""><img src="images/Email icon.png" alt="Gmail"></a></li>
+                <li><a href="https://github.com/BenMKT/Portfolio"><img src="images/Github icon.png" alt="github"></a></li>
+                <li><a href=""><img src="images/Peace icon.png" alt="site"></a></li>
+            </ul>
+        </article> 
```

Comment:
Kindly make sure that the container of the section is a `section` tag to follow HTML best practices and good semantics. 😄

Reply:
> Kindly make sure that the container of the section is a `section` tag to follow HTML best practices and good semantics. 😄

- Changed `article` tag into `section` tag.